js/main.js

```diff
@@ -1,2 +1,44 @@
 /* global data */
 /* exported data */
+
+var urlInput = document.querySelector('.photoUrl');
+var imgSrc = document.querySelector('.placeholder-img');
+urlInput.addEventListener('input', function (event) {
+
+  var imgUrl = event.target.value;
+  imgSrc.setAttribute('src', imgUrl);
+
+});
+
+var submitButton = document.getElementById('codeform');
```

Comment:
This variable is improperly named. It doesn't contain the submit button, it contains the `form` element